jira/client.py

```diff
@@ -763,6 +763,18 @@ def find(self, resource_format, ids=None):
         resource.find(ids)
         return resource
 
+    def get_changelogs_from_issue(self, issueID_or_key):
```

Comment:
these functions do not follow the style of the rest of the class.

Here we could use `self._get_json()` with the default `base` argument, the next function can use the argument `base=AGILE_BASE_URL`